packages/expo-updates/ios/EXUpdates/ReloadScreen/ReloadExceptions.swift

```diff
@@ -0,0 +1,7 @@
+import ExpoModulesCore
+
+final class ReloadOverlayException: Exception {
```

Comment:
```suggestion
internal final class ReloadOverlayException: Exception {
```

nit: explicit internal